### PR TITLE
New package: ArminOsaj&SamuelSchiegg.AutoDarkMode version 10.4.1.1

### PR DESCRIPTION
**Pull request by nett00n**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137263)

### CODE REVIEW

**Comment by nett00n** [comment]
https://github.com/microsoft/winget-pkgs/issues/137261

**Comment by nett00n** [comment]
@microsoft-github-policy-service agree

File: manifests/a/ArminOsaj&SamuelSchiegg/AutoDarkMode/10.4.1.1/ArminOsaj&SamuelSchiegg.AutoDarkMode.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: ArminOsaj&SamuelSchiegg.AutoDarkMode
+PackageVersion: 10.4.1.1
+InstallerType: inno
```
Comment (Exorcism0666) [inline manifests/a/ArminOsaj&SamuelSchiegg/AutoDarkMode/10.4.1.1/ArminOsaj&SamuelSchiegg.AutoDarkMode.installer.yaml:6]:
```suggestion
InstallerType: inno
InstallerSwitches:
  Silent: /VERYSILENT
  SilentWithProgress: /SILENT
```

**Comment by stephengillie** [comment]
Duplicate of https://github.com/microsoft/winget-pkgs/pull/132576

**Pull request closed without merging**